<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MirrorWindow</class>
 <widget class="QMainWindow" name="MirrorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>artm's mirror</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="Mirror::CompositeView" name="composite">
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>122</height>
       </size>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCapture"/>
    <addaction name="separator"/>
    <addaction name="actionInput"/>
    <addaction name="actionGreyscale"/>
    <addaction name="actionDownsampled"/>
    <addaction name="actionContrast"/>
    <addaction name="separator"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCapture"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>218</width>
     <height>116</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QFormLayout" name="formLayout">
     <property name="margin">
      <number>2</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Subsampling</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="scaleSelector">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>none</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>x 1/2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>x 1/4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>x 1/8</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Filter stage</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="stageSelector"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionCapture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
   <property name="toolTip">
    <string>Start/Stop video capture</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionInput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionGreyscale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Greyscale</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionDownsampled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Downsampled</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionContrast">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Contrast++</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Mirror::CompositeView</class>
   <extends>QGraphicsView</extends>
   <header>compositeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
